From 7414ceb6d6cd272ed9921af3d9d3537f8de4a294 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Thu, 7 Dec 2017 15:31:31 +0200
Subject: [PATCH 052/767] arm64: dts: ls1046a: disable Ethernet nodes 0 and 1

The Ethernet nodes 0 and 1 aren't present on LS1046A RDB platforms.
Remove the nodes in order to avoid error messages at boot time.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
index 87f598c59794..f84e49ba3cb4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
@@ -65,6 +65,12 @@
 };
 
 &fsldpaa {
+	ethernet@0 {
+		status = "disabled";
+	};
+	ethernet@1 {
+		status = "disabled";
+	};
 	ethernet@9 {
 		compatible = "fsl,dpa-ethernet";
 		fsl,fman-mac = <&enet7>;
-- 
2.17.0

